.github/workflows/lint.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       timeout: 120
       runner: linux.2xlarge
       docker-image: pytorch-linux-jammy-cuda11.8-cudnn8-py3.9-linter
-      fetch-depth: 1
+      fetch-depth: 0
```

Comment:
Do you mind adding a comment here, why it is needed? so we would not make the same mistake again?